includes/ConfigDefaults.php

```diff
@@ -102,7 +102,8 @@
     'crypto_crypt_name' => "AES-CBC", // The encryption algorithm used
     'crypto_hash_name' => "SHA-256", // The hash used to convert password to hashencryption_enabled
 
-    'crypto_gcm_max_file_size' => 68719476720, // 16 * (2^32 - 1)   or roughly a little under 64gb.
+    'crypto_gcm_max_file_size'  => 4294967296 * 5 * 1024 * 1024, // 4294967296 times 5 MB or roughly 16384 tb.
```

Comment:
Sorry about that, the global default is 5mb. I think I was testing 4mb locally because a site was using that specific value. The default if not set is 5mb as shown above in ConfigDefaults.php (upload_chunk_size). The upload_crypted_chunk_size has a value based on upload_chunk_size and padding, and must be correct as per the assertion I added in config loading at https://github.com/filesender/filesender/blob/master/classes/utils/Config.class.php#L290

In general, values for chunk size from 0.25mb through to 300mb are likely to be sane depending on what network conditions are offered and what backend is used for storage. Since filesender commits data at the filesender chunk level, small chunks allow upload resumes to not loose as much data. Larger, huge chunks are possible if the storage is backed by cloud rather than local storage on the server. The 5mb is a happy medium offering reasonably low overhead when the network is reasonable and the server has control over the backend storage used (eg. on the LAN).

I added a new crypto_gcm_max_chunk_count to be 2^32-1 to stop overflow on number of chunks. IIRC this is also handled elsewhere. The crypto_gcm_max_chunk_count is force set in the config loading code so the user can not mess with it.

https://github.com/filesender/filesender/pull/646/files#diff-8fa7bbec2dea5678ac5485cf0185aa23R282

That is then checked when an upload is attempted to be setup

https://github.com/filesender/filesender/pull/646/files#diff-6afcec4cdb04be9dee1fc62bf3eca799R524

I also recalculate crypto_gcm_max_file_size in config loading to make sure really silly values for tiny chunks like 20 bytes will have impact on how large of a file could be handled by such a system. The configdefualts are just the defaults, in the code linked below the default is used, the config for the site has been loaded and then I recheck some constraints and update some config settings taking into account the current site's config.php settings.

https://github.com/filesender/filesender/pull/646/files#diff-8fa7bbec2dea5678ac5485cf0185aa23R280